lib/make/makefile.raxml-ng

```diff
@@ -36,38 +36,41 @@ MAKECALL_RAXML_NG             = $(MAKECALL_RAXML_NG_VERSION); $(MAKECALL_RAXML_N
 
 # Nucleotide targets
 %.GTRG8F.raxml.rba: %.alnfna
-	raxml-ng --parse --msa $< --model GTR+G8+F --prefix $(basename $(basename $@))
+        raxml-ng --parse --msa $< --model GTR+G8+F --prefix $(basename $(basename $@))
```

Comment:
From here, and all the way down I believe, targets belong in `makefile.raxml-ng`. Do any modifications you need in that, and make a separate PR.
That you have included this here indicates that you don't know that you can `include` more than one library makefile in your `Makefile`.

Reply:
The changes are in the makefile.raxml-ng, I just pushed both files at once and I guess thats also why the pull request included it in one request. Here it also says that this part is from [lib/make/makefile.raxml-ng]